SDCTA/src/components/Body.tsx

```diff
@@ -0,0 +1,56 @@
+import landscape from '../Images/unsplash_dsV4yvL5UJk.jpg';
+import EdVis from '../Images/Graph.jpg';
+import './body.css'
+import EducationVis from '../Images/Education_Standin.png';
+import HomelessnessVis from '../Images/Homelessness_Standin.png';
+import MunicipalVis from '../Images/Municipal_Standin.png'
+
+
+export const Body = () => {
+	return (
+		<><div>
```

Comment:
Similar to comment below, we probably don't want the fragment to have the same alignment as the div because then its harder to tell that the fragment is actually the root, i.e. the top level tag.